_pages/model-deployment/mxnet.md

```diff
@@ -38,7 +38,7 @@ After training your MXNet model, you'll want to save the model and weights so yo
 ### Create a Data Collection
 Host your data where you want and serve it to your model with Algorithmia's <a href="http://docs.algorithmia.com/">Data API</a>.
 
-In this guide we'll use Algorithmia's <a href="{{site.baseurl}}/data/hosted/">Hosted Data Collection</a>, but you can host it in <a href="{{site.baseurl}}/data/s3/">S3</a> or <a href="{{site.baseurl}}/data/dropbox/">Dropbox</a> as well. Alternatively, if your data lies in a database, [check out]({{site.baseurl}}/data/dynamodb/) how we connected to a DynamoDB databasee.
+In this guide we'll use Algorithmia's <a href="{{site.baseurl}}/data/hosted">Hosted Data Collection</a>, but you can host it in <a href="{{site.baseurl}}/data/s3">S3</a> or <a href="{{site.baseurl}}/data/dropbox">Dropbox</a> as well. Alternatively, if your data lies in a database, [check out]({{site.baseurl}}/data/dynamodb/) how we connected to a DynamoDB databasee.
```

Comment:
⚠️ Looks like `/data/dynamodb/` is still trailing 👹

Reply:
I think it might be because there weren't any quotes, so could have been missed in search-replace? Might be worth scanning for any other occurrences of `/)`.

Reply:
Excellent catch. The regex I used looked was similar to `/href=".+/"/` but I didn't consider markdown style links. Searching for those now.

Reply:
This is updated.